rebar.config

```
@@ -2,11 +2,13 @@
 
 {deps, [{backoff, "~> 1.1.6"},
         {pg_types, "~> 0.4.0"},
-        {opentelemetry_api, "~> 1.0"}]}.
+        {opentelemetry_api, "~> 1.2"},
+        {opentelemetry, "~> 1.3"}]}.
```

```suggestion
        {opentelemetry_api, "~> 1.3"}]}.
```

The `opentelemetry` package is only for releases and not to be included in a library.